### PR TITLE
Step-2 Andrei: ClientVendorDTO, Controller, Service and Repo

### PR DESCRIPTION
**Pull request by AndreSelv**

### CODE REVIEW

File: src/main/resources/templates/clientvendor/client-vendor-edit.html [hunk 1]
```diff
@@ -55,31 +55,31 @@
...
                                                             </div>
                                                             <div class="form-group">
                                                                 <div class="controls">
                                                                     <label for="users-edit-state">State</label>
-                                                                    <input type="text" id="users-edit-state"
-                                                                           class="form-control" placeholder="etate"
-                                                                           value="state" required
+                                                                    <input type="text" id="users-edit-state" th:field="*{state}"
+                                                                           class="form-control" placeholder="State"
+                                                                           value="Newyork" required
```
Comment (VitalySlabada) [inline src/main/resources/templates/clientvendor/client-vendor-edit.html:82]: Looks like in this page there are couple values hardcoded,
do we need to add "TODO" to come back to it later ?

File: src/main/resources/templates/clientvendor/client-vendor-edit.html [hunk 2]
```diff
@@ -90,47 +90,55 @@
...
                                                             <div class="form-group">
                                                                 <div class="controls">
-                                                                    <label for="users-edit-company">Phone</label>
-                                                                    <input type="text" id="users-edit-company"
+                                                                    <label for="users-edit-phonenumber">Phone</label>
+                                                                    <input type="text" id="users-edit-phonenumber"
+                                                                           th:field="*{phone}"
                                                                            class="form-control"
-                                                                           placeholder="phone number">
+                                                                           placeholder="97341578663"
```
Comment (VitalySlabada) [inline src/main/resources/templates/clientvendor/client-vendor-edit.html:97]: same as above - add todo?

File: src/main/java/com/example/accountingapp/controller/ClientVendorController.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import com.example.accountingapp.service.ClientVendorService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+@RequestMapping("/client-vendor")
+public class ClientVendorController {
+
```
Comment (VitalySlabada) [inline src/main/java/com/example/accountingapp/controller/ClientVendorController.java:13]: should default page be "list" for client-vendor endpoint ?

**Review by VitalySlabada — APPROVED** [review]
nice job, couple minor comments

File: src/main/resources/templates/clientvendor/client-vendor-list.html [hunk 4]
```diff
@@ -49,32 +49,36 @@
...
+                                                        <td  th:text="${client.state.getStateCode}"></td>
+                                                        <td  th:text="${client.type}">
+                                                        <td  th:text="${client.address + ' ' + client.state.getStateCode +' '+ client.zipCode}">
 
                                                             <span class="badge bg-light-success"
 
                                                             ></span>
                                                         </td>
-<!--                                                        <td><span class="badge bg-light-success">Active</span></td>-->
+                                                        <td><span class="badge bg-light-success">Active</span></td>
```
Comment (josephum) [inline src/main/resources/templates/clientvendor/client-vendor-list.html]: This value is fixed to Active. Should be Active or Not Active depending on enabled.

Comment (josephum) [reply src/main/resources/templates/clientvendor/client-vendor-list.html]: You can include another row in the table with disable and try to show it

**Review by josephum — CHANGES_REQUESTED** [review]

**Review by josephum — APPROVED** [review]

**Pull request merged**